prow/cmd/deck/static/user-script.js

```diff
@@ -41,6 +41,10 @@ function loadUserDashboard() {
         });
     });
     const container = document.querySelector("#main-container");
+    if (!userData.PullRequests || userData.PullRequests.length === 0) {
+        container.appendChild(document.createTextNode("You don't have any open PR"));
```

Comment:
I would say "No open PRs found". The current text will be very frustrating if the data isn't loading correctly :-)

Reply:
done